browser_tests/nodeSearchBox.spec.ts

```diff
@@ -82,10 +82,14 @@ test.describe('Node search box', () => {
   test('Has correct aria-labels on search results', async ({ comfyPage }) => {
     const node = 'Load Checkpoint'
     await comfyPage.doubleClickCanvas()
-    await comfyPage.searchBox.fillAndSelectFirstNode(node)
```

Comment:
`fillAndSelectFirstNode` do the click in the first dropdown item, which will cause the searchbox to be dismissed. This will cause issue for the subsequent check of dropdown element, so change to inline those selection logic.